CMakeLists.txt

```diff
@@ -147,6 +147,10 @@ INCLUDE(${KOKKOS_SRC_PATH}/cmake/kokkos_tribits.cmake)
 # Check the environment and set certain variables
 # to allow platform-specific checks
 INCLUDE(${KOKKOS_SRC_PATH}/cmake/kokkos_check_env.cmake)
+
+#
+KOKKOS_OPTION(INSTALL_TESTING OFF BOOL "Whether to build tests and examples against installation")
```

Comment:
What this is doing effectively is to disable compiling the library and just run the tests.
Hence, I would rather name it `DISABLE_COMPILING` or similar and make sure `Kokkos_ENABLE_TESTS=ON`.